moveit_ros/moveit_servo/src/cpp_interface_demo/pose_tracking_demo.cpp

```diff
@@ -78,46 +82,59 @@ class StatusMonitor
  */
 int main(int argc, char** argv)
 {
-  ros::init(argc, argv, LOGNAME);
-  ros::NodeHandle nh("~");
-  ros::AsyncSpinner spinner(8);
-  spinner.start();
+  rclcpp::init(argc, argv);
+  rclcpp::Node::SharedPtr node = rclcpp::Node::make_shared("pose_tracking_demo");
+
+  rclcpp::executors::SingleThreadedExecutor executor;
+  executor.add_node(node);
+  std::thread([&executor]() { executor.spin(); }).detach();
+
+  moveit_servo::ServoParametersPtr servo_parameters;
+  servo_parameters = std::make_shared<moveit_servo::ServoParameters>();
+  if (!moveit_servo::readParameters(servo_parameters, node, LOGGER))
+  {
+    RCLCPP_ERROR_STREAM(LOGGER, "Could not get servo parameters!");
+    exit(EXIT_FAILURE);
+  }
 
   // Load the planning scene monitor
   planning_scene_monitor::PlanningSceneMonitorPtr planning_scene_monitor;
-  planning_scene_monitor = std::make_shared<planning_scene_monitor::PlanningSceneMonitor>("robot_description");
+  planning_scene_monitor = std::make_shared<planning_scene_monitor::PlanningSceneMonitor>(node, "robot_description");
   if (!planning_scene_monitor->getPlanningScene())
   {
-    ROS_ERROR_STREAM_NAMED(LOGNAME, "Error in setting up the PlanningSceneMonitor.");
+    RCLCPP_ERROR_STREAM(LOGGER, "Error in setting up the PlanningSceneMonitor.");
     exit(EXIT_FAILURE);
   }
 
+  planning_scene_monitor->providePlanningSceneService();
   planning_scene_monitor->startSceneMonitor();
   planning_scene_monitor->startWorldGeometryMonitor(
       planning_scene_monitor::PlanningSceneMonitor::DEFAULT_COLLISION_OBJECT_TOPIC,
       planning_scene_monitor::PlanningSceneMonitor::DEFAULT_PLANNING_SCENE_WORLD_TOPIC,
       false /* skip octomap monitor */);
-  planning_scene_monitor->startStateMonitor();
+  planning_scene_monitor->startStateMonitor(servo_parameters->joint_topic);
+  planning_scene_monitor->startPublishingPlanningScene(planning_scene_monitor::PlanningSceneMonitor::UPDATE_SCENE);
+
+  rclcpp::sleep_for(std::chrono::seconds(5));
```

Comment:
Add comment to figure out later how to wait for the planning scene monitor to be ready.